kiva/ui
# VUE-223 KvIcon: dynamically inline most icons, sprite common icons

Follow up from https://github.com/kiva/ui/pull/1541

This PR moves the majority of icon files out of the global icon sprite and into a folder where we dynamically import them only in the components which use them. This dramatically reduces the DOM nodes on all pages from ~465 to ~25 from the sprite.

- Icons which are used on most pages are kept in the sprite.
- Icons which I couldn't find a reference to in the UI repo have been removed.
- Some folder reorganization
- While testing icons if I noticed a <span or similar with an @click event on it I changed it to a <button for accessibility purposes
- After struggling with several icons not working correctly with webpack-svgstore-plugin which makes the sprite I ended up forking it and updating the SVGO dependency. That seems to have solved the issues. I have a PR up for them, hopefully it'll be resolved soon.

To test:
pull down, 
run npm install,
browse to some pages you've worked on, make sure icons look correct.
npm run storybook to see the available icons
